### PR TITLE
install will add to local store (from S3) when requested plugin not p…

### PR DESCRIPTION
**Pull request by mpopetcmed**

### CODE REVIEW

**Review by petrem — CHANGES_REQUESTED** [review]

File: encapsia_cli/plugins.py [hunk 1]
```diff
@@ -116,6 +130,8 @@ def _create_install_plan(candidates, installed, local_store, force_install):
         else:
             current_version = ""
             action = "install"
+        if will_get_from_s3 and action != "skip":
+            action = "download from s3 and " + action
```
[the remark this answers is not in the document]
Comment (mpopetcmed) [reply encapsia_cli/plugins.py:134]: I think it's important to have the download step described in the plan before user approving.

Tests can be updated, after all

[the remark this answers is not in the document]
Comment (petrem) [reply encapsia_cli/plugins.py:134]: We already add to local store, without it being mentioned in the plan, for individually listed plugins as files and when installing all available buckets. So either way, it should be uniform (show just "install" or "add and install" -- in the first case mentioned about it  is not from S3 )

I've checked a bit, I did not find tests using this output in various plugins, at a brief search. So hopefully such a change wouldn't break existing tests. Updating them can be problematic.

[the remark this answers is not in the document]
Comment (petrem) [reply encapsia_cli/plugins.py:134]: agreed to go ahead as is for now. we'll need to add other changes / refactoring so for now this is good enough.

**Comment by petrem** [comment]
fixes #95

**Pull request merged**